tf-modules/data-persistence/dynamo.tf

```diff
@@ -30,6 +30,7 @@ resource "aws_dynamodb_table" "access_tokens_table" {
 
   lifecycle {
     prevent_destroy = true
+    ignore_changes = [ name ]
```

Comment:
What does this do?

Reply:
In the docs, if you want to restore a table to an earlier version you have to restore to a different table name. To get that new table into your deployment you import it into Terraform with `terraform import`. When you go to redeploy our table names are hardcoded to something like `prefix-CollectionsTable`. Without this it'll try to create `prefix-CollectionsTable` even if you've imported `collectionstable-2` which will create you a new table and then you lose all your data. With this line, if you import a table with a different name it'll ignore those changes and not try to completely redeploy/recreate your table if only the name has changed.

Reply:
Hmm, that's difficult. Ordinarily, I would think that we *do want* the old table to be deleted if you have changed the table name, for example if we update the default table names in the future. But that's an edge case that may never really happen. And as you say, I don't see how we can support the backup to a new table name without this change

Reply:
If we updated table names and then deleted everyone's table and created a new one with the new table name our users would have a problem. Really I don't think we ever want to be regenerating the table since it would have data in it. We'd have to have a complex migration scenario.

Reply:
Yeah, you're right. Makes sense. Definitely a good change then 👍